helpers/networks.js

```diff
@@ -0,0 +1,22 @@
+const getNetworkNameFromId= (networks, id) => {
+  const defaultNetwork = 'devnet'
+  for (let n in networks) {
+    if (networks[n].network_id == id) {
+      return n
+    }
+  }
+  return defaultNetwork
+}
+const getNetworkId = () =>
+        new Promise(
```

Comment:
Indent :)

Reply:
My editor did that automatically, how should I indent it? Just 2 spaces from the beginning?